src/game-views/cards/OrderCard.tsx

```diff
@@ -13,15 +13,13 @@ interface Props {
 
 const OrderCard: React.FunctionComponent<Props> = props => {
     const { wines, victoryPoints, residualIncome } = props.cardData;
-    return <div className={cx("OrderCard", props.className)}>
-        <div className="OrderCard-name">Order</div>
-        <div className="OrderCard-description">
-            <div className="OrderCard-wines">
-                {wines.map(wine => (
-                    <WineGlass key={`${wine.color}${wine.value}`} className="OrderCard-wine" color={wine.color}>{wine.value}</WineGlass>
-                ))}
-            </div>
-            <div className="OrderCard-rewards">
+    return <div className={cx("PlayerCard", "PlayerCard--order", props.className)}>
```

Comment:
can you make a `PlayerCard` component instead? the classNames should match the component name